pyproject.toml

```diff
@@ -70,6 +70,13 @@ module = [
     "trio._sync",
     "trio._tools.gen_exports",
     "trio._util",
+    "trio.testing",
+    "trio.testing._check_streams",
+    "trio.testing._checkpoints",
+    "trio.testing._memory_streams",
+    "trio.testing._network",
+    "trio.testing._sequencer",
+    "trio.testing._trio_test",
```

Comment:
can `_highlevel_generic` be added here in this PR? given that #2756 doesn't touch it

Reply:
added in 2b669c7